core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -1410,22 +1431,19 @@ trait EntityComponent {
     private def unmarshalEntity(entityRecord: EntityRecord, attributes: AttributeMap): Entity =
       Entity(entityRecord.name, entityRecord.entityType, attributes)
 
-    private def unmarshalEntities(
-      entityAttributeRecords: Seq[entityQuery.EntityAndAttributesRawSqlQuery.EntityAndAttributesResult]
+    def unmarshalEntities(
+      entityAttributeRecords: Seq[EntityAndAttributesResult]
     ): Seq[Entity] =
       unmarshalEntitiesWithIds(entityAttributeRecords).map { case (_, entity) => entity }
 
-    private def unmarshalEntitiesWithIds(
-      entityAttributeRecords: Seq[entityQuery.EntityAndAttributesRawSqlQuery.EntityAndAttributesResult]
+    def unmarshalEntitiesWithIds(
+      entityAttributeRecords: Seq[EntityAndAttributesResult]
     ): Seq[(Long, Entity)] = {
```

Comment:
I know this line wasn't changed -- but why is `Long` used here instead of something like `AnyVal`?

Reply:
This is returning a `Seq` of 2-tuples, where the tuple consists of the entity id and the `Entity` object. The entity id is the db's primary key, which is a Long.